Client/Frontend/Widgets/TextFieldTableViewCell.swift

```diff
@@ -6,69 +6,74 @@ import Common
 import UIKit
 import Shared
 
-struct TextFieldTableViewCellUX {
-    static let HorizontalMargin: CGFloat = 16
-    static let VerticalMargin: CGFloat = 10
-    static let TitleLabelFont = UIFont.systemFont(ofSize: 12)
-    static let TextFieldFont = UIFont.systemFont(ofSize: 16)
-}
-
 protocol TextFieldTableViewCellDelegate: AnyObject {
     func textFieldTableViewCell(_ textFieldTableViewCell: TextFieldTableViewCell, didChangeText text: String)
 }
 
 class TextFieldTableViewCell: UITableViewCell, ThemeApplicable {
-    let titleLabel: UILabel
-    let textField: UITextField
+    private struct UX {
+        static let HorizontalMargin: CGFloat = 16
+        static let VerticalMargin: CGFloat = 10
+        static let TitleLabelFont = UIFont.systemFont(ofSize: 12)
+        static let TextFieldFont = UIFont.systemFont(ofSize: 16)
+    }
+
+    private lazy var titleLabel: UILabel = .build()
+    private lazy var textField: UITextField = .build()
 
     weak var delegate: TextFieldTableViewCellDelegate?
 
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
-        self.titleLabel = UILabel(frame: .zero)
-        self.textField = UITextField(frame: .zero)
-
         super.init(style: style, reuseIdentifier: reuseIdentifier)
 
-        self.contentView.addSubview(self.titleLabel)
-        self.contentView.addSubview(self.textField)
-        self.textField.addTarget(self, action: #selector(onTextFieldDidChangeText), for: .editingChanged)
+        contentView.addSubviews(titleLabel, textField)
+        textField.addTarget(self, action: #selector(onTextFieldDidChangeText), for: .editingChanged)
         self.selectionStyle = .none
         self.separatorInset = .zero
+
+        setupLayout()
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
-    override func layoutSubviews() {
-        super.layoutSubviews()
-
-        titleLabel.font = TextFieldTableViewCellUX.TitleLabelFont
-        titleLabel.snp.remakeConstraints { make in
-            guard titleLabel.superview != nil else { return }
-
-            make.leading.equalTo(TextFieldTableViewCellUX.HorizontalMargin)
-            make.trailing.equalTo(-TextFieldTableViewCellUX.HorizontalMargin)
-            make.top.equalTo(TextFieldTableViewCellUX.VerticalMargin)
-        }
-
-        textField.font = TextFieldTableViewCellUX.TextFieldFont
-        textField.snp.remakeConstraints { make in
-            guard textField.superview != nil else { return }
+    private func setupLayout() {
+        titleLabel.font = UX.TitleLabelFont
+        guard let titleLabelSuperview = titleLabel.superview else { return }
```

Comment:
Can't we constraint directly to the contentView since we know that's the superview, instead of having to unwrap `.superview`?